### ISSUE TITLE
[NBI] Tree crown detection and multi-class classification (DeepForest)

### ISSUE THREAD

**Issue by camappel** [eds-book-gallery/notebooks-ideas#4]
## What is the notebook about?

The notebook will demonstrate how to use the latest version of DeepForest's [CropModel](https://deepforest.readthedocs.io/en/latest/CropModels.html) for tree crown detection and multi-class classification using pre-trained models.

It will explore the performance of DeepForest's pre-trained models on different publicly available pre-processed data: 
- Homogeneous vs heterogeneous landscape
- Small distinct tree-crowns vs large dense canopies
- High vs low resolution images
- Tiles vs entire images

Additionally, it will cover post-processing steps to evaluate and visualise the model's predictions.

## Data Science Component

- [x] Exploration
- [ ] Preprocessing
- [ ] Modelling
- [x] Post-processing
- [ ] Other (e.g. Reproducibility): 

## Submission type

- [x] Standard
- [ ] Special Issue
- [ ] Other (e.g. CI2023 Reproducibility Challenge):

## Programming language

- [x] Python
- [ ] R
- [ ] Julia
- [ ] Other:

## Checklist:

- [x] Input data, pipeline and/or model are public with license/citation
- [x] The proposed notebook reuses existing codebase
- [x] The proposed notebook uses open-source packages
- [x] The proposed notebook is associated to existing publication(s)

## Additional information 
An EDS [notebook](https://edsbook.org/notebooks/gallery/15d986da-2d7c-44fb-af71-700494485def/notebook) already exists on tree crown detection using DeepForest. This notebook is different because it focusses on the latest version of DeepForest (1.4.0), which utilises model-chaining to classify detected tree crowns into multiple classes.

### Extension
DeepForest also allows uses to [train](https://deepforest.readthedocs.io/en/latest/training.html) the models: "The prebuilt models will always be improved by adding data from the target area. In our work, we have found that even an hour’s worth of carefully chosen hand-annotation can yield enormous improvements in accuracy and precision. 5-10 epochs of fine-tuning with the prebuilt model are often adequate." [Furthermore](https://github.com/weecology/DeepForest/discussions/677 
), "the backbone model is really meant for fine-tuning based on local annotations."

As such, there is the potential to extend the notebook to include fine-tuning the model by acquiring and pre-processing aerial RGB images using drones.
